File: Content/assets_src/js/model/_usuarioClienteSite.js
/*
|--------------------------------------------------------------------------
| Model UsuarioClienteSite
|--------------------------------------------------------------------------
*/

APP.model.UsuarioClienteSite = {
    
       model: {
            IdSite : "",
            IdCliente : "",
        },
       
    
        constructor : function (_IdSite, _IdCliente)
        {
            var usuarioClienteSite = {
                IdSite : _IdSite,
                IdCliente : _IdCliente
            };
            
            return usuarioClienteSite;
        }
    
    };
    

File: Dominio/Entidade/UsuarioClienteSite.cs
namespace Dominio.Entidade
{
    public class UsuarioClienteSite
    {
        public int IdUsuarioClienteSite { get; set; }
        public int IdCliente { get; set; }
        public int IdUsuario { get; set; }
        public int? IdSite { get; set; }
        public virtual Cliente Cliente { get; set; }
        public virtual Usuario Usuario { get; set; }
        public virtual Site Site { get; set; }
    }
}
